runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -39,6 +39,12 @@ use crate::{
 };
 use inherents::{InherentIdentifier, InherentData, MakeFatalError, ProvideInherent};
 
+// In the future, we should benchmark these consts; these are all untested assumptions for now.
```

Comment:
why not adding benchmarks now ? it shouldn't be too hard with `benchmark!` macro, you can look at other pallet how they write benchmark.

But I'm not familiar with the inclusion code so I might underestimate the task.

Reply:
Promoted this to a new issue: https://github.com/paritytech/polkadot/issues/2124. It's kind of out of scope of this PR because the old numbers were essentially guesses as well.